lcx-api/adapter.js

```diff
@@ -0,0 +1,71 @@
+const { Requester, Validator } = require('@chainlink/external-adapter')
+
+const customError = (data) => {
+  return !data || !data.data || !data.data.Price
+}
+
+const customParams = {
+  symbol: false
+}
+
+const headers = {
+  'Content-Type': 'application/json',
+  'api-key': process.env.API_KEY
+}
+
+const ALL = 'all'
+const SYMBOLS = ['btc-usd', 'btc-eur', 'eth-usd', 'eth-eur']
+
+function getUrlForSymbol (symbol) {
```

Comment:
This function returns an array in the case of `ALL` and a string in case of any symbol, which is inconsistent and prone to bugs. Please split this into two functions.